plugin-gestures/src/main/java/com/mapbox/maps/plugin/gestures/GestureDelegateExt.kt

```diff
@@ -0,0 +1,133 @@
+@file:JvmName("MapboxMapUtils")
```

Comment:
Why is it `MapboxMapUtils`, all functions seem to be gestures related

Reply:
It's extension functions for MapboxMap, we use these method like this: `mapboxMap.addOnMapClickListener(this)`

Reply:
There's smth wrong with such logic as similar animation functions (e.g. `flyTo`) are located in `CameraAnimationsExt.kt` for now but they do the same thing actually allowing calling `mapboxMap.flyTo(...)`

Reply:
Changed to `GesturesUtils`